tests/unity/wrap_test/src/fns/fns.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/*
+ * DO NOT FORMAT THIS FILE!
+ *
+ * It contains function declarations that is used to validate the test wrapper
+ * scripts (func_name_list.py and header_parse.py) which renames the functions.
+ * The regular expression has had several bugs in the past, and the formating
+ * of the functions here are used to validate that	the fixes still work when
+ * modifying the regex.
+ * List of fixes this file tries to reproduce:
+ * https://github.com/nrfconnect/sdk-nrf/pull/1413
+ * https://github.com/nrfconnect/sdk-nrf/pull/2182
+ * https://github.com/nrfconnect/sdk-nrf/pull/3432
+ * https://github.com/nrfconnect/sdk-nrf/pull/5069
+ * https://github.com/nrfconnect/sdk-nrf/pull/5814
+ * https://github.com/nrfconnect/sdk-nrf/pull/6478
+ * https://github.com/nrfconnect/sdk-nrf/pull/6547
+*/
+
+#ifndef __FNS_H
+#define __FNS_H
+
+/* #include <toolchain/common.h> */
+
+// c++ comment
+/* c comment */
+/* c
+ * multi line comment
+ */
```

Comment:
How does this match this description:
https://github.com/nrfconnect/sdk-nrf/blob/42e3f1a5f33bd088d8ac429ce49d60fd7fff8478/tests/unity/wrap_test/src/fns/fns.h#L10-L11

Those different comments styles are testing that comments are correctly ignored when parsing the file, but I cannot be the description in `fns.h` see why they belong here.

So either a comment explaining why those comments are in this file, or update the description of the file.

Btw: 👍 for a very good description of the intention of this file.

Reply:
Good point. Have updated the comment. Seems to be related to this #2182. Which is not properly tested. Will create a test to validate it too.